src/main/java/driver/Main.java

```diff
@@ -3,22 +3,25 @@
 import entities.StateTracker;
 import fworks.da.PostgresAccessManager;
 import ia.gateways.DatabaseAccessGateway;
+import ui.MainFrame;
 
 import java.sql.SQLException;
 
 public class Main {
 
     public static void main(String[] args) {
-        StateTracker currentState = new StateTracker();
-        System.out.print("====    U of T Exam Centre     ===\n");
-        System.out.print("====  CSC 207 Course Project   ===");
+        new MainFrame();
 
-        try {
-            DatabaseAccessGateway db = new PostgresAccessManager("localhost", 5432,
-                    "postgres", "exam_centre", "postgrespw", false);
-        } catch (SQLException e) {
-            throw new RuntimeException(e);
-        }
+//        StateTracker currentState = new StateTracker();
```

Comment:
Although it is not currently used since most use case controllers aren't implemented, the initialisation of the PostgresAccessManager will be necessary when injecting gateways into use cases. It can remain commented out for now though.